tests/components/manual/test_alarm_control_panel.py

```diff
@@ -1435,16 +1436,16 @@ async def test_arm_away_after_disabled_disarmed(hass):
     await common.async_alarm_arm_away(hass, CODE)
 
     state = hass.states.get(entity_id)
-    assert STATE_ALARM_PENDING == state.state
-    assert STATE_ALARM_DISARMED == state.attributes["pre_pending_state"]
-    assert STATE_ALARM_ARMED_AWAY == state.attributes["post_pending_state"]
+    assert STATE_ALARM_ARMING == state.state
+    assert STATE_ALARM_DISARMED == state.attributes["previous_state"]
+    assert STATE_ALARM_ARMED_AWAY == state.attributes["next_state"]
 
     await common.async_alarm_trigger(hass, entity_id=entity_id)
 
     state = hass.states.get(entity_id)
-    assert STATE_ALARM_PENDING == state.state
-    assert STATE_ALARM_DISARMED == state.attributes["pre_pending_state"]
-    assert STATE_ALARM_ARMED_AWAY == state.attributes["post_pending_state"]
+    assert STATE_ALARM_ARMING == state.state
+    assert STATE_ALARM_DISARMED == state.attributes["previous_state"]
+    assert STATE_ALARM_ARMED_AWAY == state.attributes["next_state"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5411564)